lib/tasks/npm-cache-clean.js

```diff
@@ -0,0 +1,12 @@
+'use strict';
+
+// Runs `npm cache clean` in cwd
+
+var NpmTask = require('./npm-task');
+
+module.exports = NpmTask.extend({
+    command: 'cache clean',
+    startProgressMessage: 'Cleaning Npm cache',
+    completionMessage: 'Npm cache cleaned successfully',
```

Comment:
Prefer `npm` over `Npm`, as that's how npmjs.org references it (in all lower case)